### PR TITLE
Feature: Load image as background

### PR DESCRIPTION
**Pull request by kaamaurice**
## Brief description
added : Loader for IMage family as Baground image for camera.

### CODE REVIEW

File: openpype/hosts/blender/plugins/load/load_image.py [hunk 1]
```diff
@@ -64,3 +61,38 @@ def process_asset(self, context: dict, *args, **kwargs) -> bpy.types.Object:
...
+                    obj for obj in bpy.context.scene.objects
+                    if obj.type == "CAMERA"
+                ),
+                None,
+            )
+
+        if camera:
+            camera.data.show_background_images = True
+            bkg_img = camera.data.background_images.new()
+            bkg_img.image = img
```
Comment (Tilix4) [inline openpype/hosts/blender/plugins/load/load_image.py:95]: Please add comments to sum steps up.

**Review by Tilix4 — CHANGES_REQUESTED** [review]

File: openpype/hosts/blender/plugins/load/load_image.py [hunk 2]
```diff
@@ -59,8 +59,59 @@ def process_asset(self, context: dict, *args, **kwargs) -> bpy.types.Object:
...
+            camera = next(
+                (
+                    obj for obj in bpy.context.scene.objects
+                    if obj.type == "CAMERA"
+                ),
+                None,
+            )
+
+        assert camera, "No camera found!"
+
```
Comment (Tilix4) [inline openpype/hosts/blender/plugins/load/load_image.py:106]:
```suggestion

        # Add movie as background image
```

File: openpype/hosts/blender/plugins/load/load_image.py [hunk 3]
```diff
@@ -31,17 +27,21 @@ def process_asset(self, context: dict, *args, **kwargs) -> bpy.types.Image:
...
         img.name = plugin.asset_name(asset, subset)
 
+        if libpath.endswith((".mov", "mp4")):
+            img.source = 'MOVIE'
+
         return img
 
 
 class ReferenceLoader(ImageLoader):
     """Load image as Reference."""
```
Comment (Tilix4) [inline openpype/hosts/blender/plugins/load/load_image.py:37]: Could you please explain a bit more the behaviour please? This way we can understand the difference between the loaders.

File: openpype/hosts/blender/plugins/load/load_image.py [hunk 4]
```diff
@@ -1,20 +1,16 @@
...
 import bpy
 
 from openpype.hosts.blender.api import plugin
-from openpype.hosts.blender.api.pipeline import AVALON_PROPERTY
-from openpype.pipeline.load.utils import get_representation_path
+from openpype.hosts.blender.api.lib import get_selection
 
 
 class ImageLoader(plugin.AssetLoader):
     """Load image in Blender."""
```
Comment (Tilix4) [inline openpype/hosts/blender/plugins/load/load_image.py:10]: I bet it is only in the image viewer?

**Review by Tilix4 — CHANGES_REQUESTED** [review]

File: openpype/hosts/blender/plugins/load/load_image.py (hunk 4 above)
Comment (kaamaurice) [reply openpype/hosts/blender/plugins/load/load_image.py]: doc updated

File: openpype/hosts/blender/plugins/load/load_image.py (hunk 3 above)
Comment (kaamaurice) [reply openpype/hosts/blender/plugins/load/load_image.py]: doc updated

File: openpype/hosts/blender/plugins/load/load_image.py (hunk 2 above)
Comment (kaamaurice) [reply openpype/hosts/blender/plugins/load/load_image.py:122]: comment updated

**Review by Tilix4 — APPROVED** [review]
LGTM

**Pull request merged**